libs/tree-sitter-wing/package.json

```diff
@@ -20,7 +20,14 @@
   },
   "scripts": {
     "test": "npx tree-sitter-cli test",
-    "test:update": "npx tree-sitter-cli test --update"
+    "test:update": "npx tree-sitter-cli test --update",
```

Comment:
I should have known there was an --update flag... the last time I updated the grammar I was updating the snapshots by hand 😅